frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -0,0 +1,381 @@
+import path from "path";
+import { USERS, restore, signOut } from "__support__/cypress";
+
+const admin = USERS.admin;
+const new_user = {
+  first_name: "Barb",
+  last_name: "Tabley",
+  username: "new@metabase.com",
+};
+
+describe("metabase-smoketest > admin", () => {
+  before(() => restore("blank"));
+
+  it("should set up Metabase", () => {
+    // ****************
+    // ***  ADMIN   ***
+    // *****************
+
+    // This is a simplified version of the "scenarios > setup" test
+    cy.visit("/");
+    cy.contains("Welcome to Metabase");
```

Comment:
`contains` (built in to Cypress) and `findByText` (part of `@testing-library/cypress` are pretty similar. Is there a reason you're using both?

I would prefer sticking with `testing-library` functions where possible but if there's a good reason to use `contains` that's fine.

`findByText` is a little more strict, which I think is generally a good thing, By default it only matches exact text (unless you pass `{ exact: false }` as the second argument) and will fail if there's more than one match (unless you use `findAllByText`).

Reply:
As you noticed, I used both so I could more quickly determine where I am navigating the site and where I am simply making assertions. 

I don't have strong feelings about the distinctions, so I'll switch over to findByText and findAllByText. 

Thank you for explaining your preference!

Reply:
FYI some of the preferences are discussed in https://github.com/metabase/metabase/wiki/E2E-Tests-with-Cypress but it's not entirely up to date.